mod_pbxproj/mod_pbxproj.py

```diff
@@ -719,11 +798,16 @@ def get_groups_by_os_path(self, path):
 
         return groups
 
-    def get_build_phases(self, phase_name):
-        phases = [p for p in self.objects.values() if p.get('isa') == phase_name]
+    def get_build_phases(self, phase_isa, phase_name = None):
```

Comment:
I changed the old first parameter phase_name to  phase_isa，because it's value is “PBX*”，corresponding to property isa，not name.
the second parameter phase_name is corresponding to property name。
so this change won't break the backwards compatibity.